internal/common/test_configuration.go

```diff
@@ -239,6 +239,41 @@ func GenerateTestDefaultInventory() string {
 	return string(b)
 }
 
+func GenerateTestInventoryWithVirtualInterface() string {
+	inventory := &models.Inventory{
+		Interfaces: []*models.Interface{
+			{
+				Name: "eth0",
+				IPV4Addresses: []string{
+					"1.2.3.4/24",
```

Comment:
nit, but you could use addresses from the subnets specifically designated for use in tests and documentation here.

They are `192.0.2.0/24`, `198.51.100.0/24`, and `203.0.113.0/24`
https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Modified it! Thank you!